portal/date_tools.py

```diff
@@ -16,13 +16,17 @@ def as_fhir(obj):
     if hasattr(obj, 'as_fhir'):
         return obj.as_fhir()
     if isinstance(obj, datetime):
-        # Make SURE we only communicate unaware or UTC timezones
+        # Make SURE we only communicate UTC timezone aware objects
         tz = getattr(obj, 'tzinfo', None)
         if tz and tz != pytz.utc:
             current_app.logger.error("Datetime export of NON-UTC timezone")
-        return obj.strftime("%Y-%m-%dT%H:%M:%S%z")
+        if not tz:
```

Comment:
Is it dangerous to presume UTC if it hasn't been explicitly passed?

Reply:
We convert to UTC if given a different timezone.  We're documented to expect UTC.  I think it'd raise problems to enforce a required tz input, but that could be done.

Reply:
>We convert to UTC if given a different timezone

Sounds good, does it make sense for that to generate an error (log) message though?